styles/styles.css

```
@@ -642,4 +642,10 @@ body.docs-template main .section.dark .default-content-wrapper {
   border-color: var(--spectrum-blue);
 }
 
-
+main .separator
```

Make it clear that this is a `.section` rule:
```suggestion
main .section.separator
```